### DISCUSSION TITLE
After 1.15.5 a fully working project suddenly does not build anymore but I have no time to reproduce right now

### DISCUSSION THREAD

**Discussion by STR1234** [sveltejs/kit#9687] [General]
I wanted to file a bug report but I do not have enough information to provide a reproduction (and no time to do so) right now.
I also feel like, I might not fully understand what the new change (check) is actually doing behind the scenes as to mark it
as a bug right now.

My question is:
When I have a previously fully working project with only one +layout.svelte file in the src/routes/ folder, is sveltekit then supposed to error out with the message: 

**Multiple universal layout module files found in src/routes/ : +layout.js and +layout.js**

I would understand, if I were clearly using multiple +layout files but I am not and also not importing any other layout files in the +layout.svelte.
Otherwise I am not aware of any module that might load anything like an additional layout file which might be conflicting here...if that is a possibility.
Maybe somebody has any insights in this or faced the same problem after updating to sveltekit version 1.15.5.
I would be glad for more information on how the new check functions, what might be triggering this message.

Additionally: I just tried to build the project to look into the built artifacts for any multiple layout files probably generated but the build also errors before any files are generated with the same error message.

**Comment by geoffrich** [comment]
You may have seen it, but this is the PR where this error was introduced: https://github.com/sveltejs/kit/pull/9590. It was intended to catch invalid cases like having both a `+layout.js` and a `+layout.ts` file for the same route.

If that isn't the case for your project, then a minimal reproducible would be appreciated if you have time -- if you only have a single `+layout.svelte` file in the root, it sounds like a bug.

**Reply by STR1234** [reply]
Okay thank you for your answer - I am going to look into building a reproduction as soon as possible :)

**Comment by maxzeroedge** [comment]
Just wanted to add a quick note. I am also facing a similar issue. Apparently, there is some problem between the houdini plugin and latest @sveltejs/kit (worked fine in 1.15.0, but fails in 1.15.7)
Removing houdini() from plugins in vite.config lets the build succeed. 

Here's a stackblitz for example: https://stackblitz.com/edit/sveltejs-kit-template-default-jjttza

**Reply by geoffrich** [reply]
Thanks for the minimal repro - I opened https://github.com/sveltejs/kit/issues/9735 with it

**Reply by geoffrich** [reply]
Actually, seems like this was fixed on the Houdini end. See the issue I linked for details.

**Reply by STR1234** [reply]
Wohooo great news :) 
Thanks @maxzeroedge you are amazing! 
Thank you that you made the repro - I had no time to do it/investigate - big release coming up on a project right now 😅 

Glad to hear that the houdini team already fixed the issue 🙌  
Thank you for your time @geoffrich :)
